packages/css/spinner.css

```diff
@@ -1,42 +1,58 @@
 .ds-spinner {
   --dsc-spinner-background: var(--ds-color-neutral-surface-default);
-  --dsc-spinner-stroke: var(--ds-color-accent-base-default);
+  --dsc-spinner-stroke: var(--ds-color-neutral-border-default);
+  --dsc-spinner-size: 40px;
```

Comment:
Should this size be set in rem/ds-spacing? ☺️

Reply:
I just copied over what was in the `sizeMap`, and did not check if we have sizing variables for this.
I'll check and update accordingly.

Reply:
Had to do some math to get the desired sizes, but it now uses sizing variables for sizes

Reply:
Nice! saw the `79px` was quite funky- should we double check with design before committing such a complex calc? ☺️

Reply:
Sure, I just copied what was already there. So unless anything has changed in figma since this was created, it is for now correct. If I remember correctly they can't use variables for width and height on svg, but it would be nice if the sizes were at least divisible by 4